templates/decrypt-tls-assets

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+rkt run \
```

Comment:
This is just to be able to use the `aws` command line tool. The encrypted keys are in `/etc/kubernetes/ssl` and the corresponding decrypted keys will be in the same directory.